pkg/continuoustest/write_read_series.go

```diff
@@ -236,24 +255,25 @@ func (t *WriteReadSeriesTest) runRangeQueryAndVerifyResult(ctx context.Context,
 	if err != nil {
 		t.metrics.queriesFailedTotal.Inc()
 		level.Warn(logger).Log("msg", "Failed to execute range query", "err", err)
-		return
+		return fmt.Errorf("failed to execute range query: %w", err)
```

Comment:
I would prefer to use `errors.Wrap()` instead of `fmt.Errorf()` to wrap errors. That's the pattern we use across Mimir repo. Do you mind replacing it in this PR, please?